components/categories/Categories.tsx

```diff
@@ -44,7 +44,7 @@ const Categories = () => {
                 supportingText="Ranging from career growth to personal development"
             />
 
-            <Shelf title="Career Growth" link="#">
+            <Shelf title="Career Growth" link="/categories/career-growth">
```

Comment:
Hey! Just for consistency, let's camelCase the URLs (that's an example of camel case haha). So /categories/careerGrowth. May change this in the future, but yeah that's how they are right now.